marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/ingestion/bulkIngester.sjs

```diff
@@ -29,6 +29,29 @@ const consts = require('/data-hub/5/impl/consts.sjs');
 const state = fn.head(xdmp.fromJSON(endpointState));
 
 const work = fn.head(xdmp.fromJSON(workUnit));
+let collections=[];
+if(work.collections !=null){
+  collections=work.collections.split(',');
+}
+
+const permissionsArray = []
+
+if(!work.permissions){
+  work.permissions='data-hub-common,read,data-hub-common,update'
+}
+
+const permissions = work.permissions.split(',');
```

Comment:
We need a test verifying that it's okay if permissions is not, then the documents will be added with no permissions. You'll need to use "runAsAdmin" after ingesting the data to verify that the documents exist.

We can't force Ernie to specify permissions, he's free to insert documents without any permissions.